MTA 2072 rework assessment module (#806)
windup/windup-documentation: docs/topics/mta-web-assessment-module.adoc

. Click *Application Inventory* in the left menu bar. A list of all the available applications appears in the main pane. 
. Click `edit` {edit} to open the application settings, and review the application settings. See procedure above for a list of application settings.

:_content-type: PROCEDURE
[id="mta-assessment-run_{context}"]
== Running an assessment on an application

.Prerequisites

* Log into a server running {ProductShortName}

.Procedure

image::mta-assessment-run-01.png[MTA application inventory]

. In the {ProductShortName} user interface, select the *Migration* working mode *.
. Click *Application Inventory* in the left menu bar. A list of all the available applications appears in the main pane. 
. Select an application from the list that appears in the main pane, click image:mta-web-icon-actions.png[actions] (three dots) at the right end of the row, and select *Assess* from the drop-down menu.

// Get updated image for MTA
image::mta-assessment-select-questionnaire-01.png[Assessment questionnaires]

[start=4]
. *Assessment Action* appears in the main pane. Select one of the questionnaires and click *Take*.

[start=5]
. *Application assessment*

> [id="mta-assessment-start-questionnaire_{context}"]
> === Opening an existing assessment questionnaire

Author: ```suggestion

```

. Click *Application Inventory* in the left menu bar. A list of all the available applications appears in the main pane. 
. Click `edit` {edit} to open the application settings, and review the application settings. See procedure above for a list of application settings.

:_content-type: PROCEDURE
[id="mta-assessment-run_{context}"]
== Running an assessment on an application

.Prerequisites

* Log into a server running {ProductShortName}

.Procedure

image::mta-assessment-run-01.png[MTA application inventory]

. In the {ProductShortName} user interface, select the *Migration* working mode *.
. Click *Application Inventory* in the left menu bar. A list of all the available applications appears in the main pane. 
. Select an application from the list that appears in the main pane, click image:mta-web-icon-actions.png[actions] (three dots) at the right end of the row, and select *Assess* from the drop-down menu.

// Get updated image for MTA
image::mta-assessment-select-questionnaire-01.png[Assessment questionnaires]

[start=4]
. *Assessment Action* appears in the main pane. Select one of the questionnaires and click *Take*.

> // Get updated image for MTA
> image::mta-assessment-select-questionnaire-01.png[Assessment questionnaires]
> 

Author: ```suggestion

```

* Assessed applications are grouped into *archetypes* with similar application characteristics.
* Assessment module integrates tightly with the application inventory.
* Assessment questionnaires can be imported and exported.
* Enhanced questionnaire capabilities:
** Conditional questions
** Application auto-tagging according to answers
** Automated answers from tags in application archetypes

In the MTA user interface, the Assessment Module appears in a separate menu item. Selecting the item shows the currently saved applications.

The {ProductName} ({ProductShortName}) assesses applications that are to be migrated, according to a set of questions relevant to the application, such as dependencies, *[more question types]*. 
// {ProductShortName} comes with a default questionnaire/s that the user can edit, customize and save as new questionnaires.

[id="mta-assessment-new-app_{context}"]
== Creating a new application

This procedure described how to create a new application in {ProductShortName} that is not based on the settings of an existing application.

.Prerequisites

* Log into a server running {ProductShortName}

.Procedure

> [MTA application inventory]image::mta-assessment-app-create-01.png

Author: ```suggestion
image::mta-assessment-app-create-01.png[MTA application inventory]
```

.. *Tags*: software tags that characterize the application (mandatory)
.. *Owner*: registered software owner from drop-down list (optional)
.. *Contributors*: contributors from drop-down list (optional)
.. *Comments*: relevant comments on the migrated application
. Click *Source code* and enter the following fields.
.. *Repository type*: *Git* or *Subversion* (mandatory)
.. *Source Repository*: URL of the repository where the software code is saved (mandatory)
.. *Branch*: application code branch in the repository (mandatory)
.. *Root path*: [get info] (optional/mandatory)
. Click *Binary* and enter the following fields.
.. *Group*: [get info] (optional/mandatory)
.. *Artifact*: [get info] (optional/mandatory)
.. *Version*: software version of the application
.. *Packaging*: [get info] (optional/mandatory)
. Click *Create*. The dialog closes and the new application appears in the list of defined applications.

[id="mta-assessment-open-app_{context}"]
== Opening an existing application

This procedure describes how to open and edit an application that has already been defined in {ProductShortName}.

.Prerequisites

* Log into a server running {ProductShortName}

.Procedure

> image::mta-assessment-apps-01.png[]

Author: ```suggestion
image::mta-assessment-apps-01.png[missing description]
```

. Click *Application Inventory* in the left menu bar. A list of all the available applications appears in the main pane. 
. Click `edit` {edit} to open the application settings, and review the application settings. See procedure above for a list of application settings.

:_content-type: PROCEDURE
[id="mta-assessment-run_{context}"]
== Running an assessment on an application

.Prerequisites

* Log into a server running {ProductShortName}

.Procedure

image::mta-assessment-run-01.png[MTA application inventory]

. In the {ProductShortName} user interface, select the *Migration* working mode *.
. Click *Application Inventory* in the left menu bar. A list of all the available applications appears in the main pane. 
. Select an application from the list that appears in the main pane, click image:mta-web-icon-actions.png[actions] (three dots) at the right end of the row, and select *Assess* from the drop-down menu.

// Get updated image for MTA
image::mta-assessment-select-questionnaire-01.png[Assessment questionnaires]

[start=4]
. *Assessment Action* appears in the main pane. Select one of the questionnaires and click *Take*.

// Get updated image for MTA

> image::mta-assessment-questions-01.png[Assessment questionnaires]

Author: ```suggestion
image::mta-assessment-select-questionnaire-01.png[Assessment questionnaires]
```

[start=5]
. *Application assessment*

> :_content-type: PROCEDURE

Author: ```suggestion

```

. Click *Binary* and enter the following fields.
.. *Group*: [get info] (optional/mandatory)
.. *Artifact*: [get info] (optional/mandatory)
.. *Version*: software version of the application
.. *Packaging*: [get info] (optional/mandatory)
. Click *Create*. The dialog closes and the new application appears in the list of defined applications.

[id="mta-assessment-open-app_{context}"]
== Opening an existing application

This procedure describes how to open and edit an application that has already been defined in {ProductShortName}.

.Prerequisites

* Log into a server running {ProductShortName}

.Procedure

image::mta-assessment-apps-01.png[]

. In the {ProductShortName} user interface, select the *Migration* working mode.
. Click *Application Inventory* in the left menu bar. A list of all the available applications appears in the main pane. 
. Click `edit` {edit} to open the application settings, and review the application settings. See procedure above for a list of application settings.

:_content-type: PROCEDURE
[id="mta-assessment-run_{context}"]
== Running an assessment on an application

.Prerequisites

* Log into a server running {ProductShortName}

.Procedure

> // Get updated image for MTA

Author: ```suggestion

```

.. *Tags*: software tags that characterize the application (mandatory)
.. *Owner*: registered software owner from drop-down list (optional)
.. *Contributors*: contributors from drop-down list (optional)
.. *Comments*: relevant comments on the migrated application
. Click *Source code* and enter the following fields.
.. *Repository type*: *Git* or *Subversion* (mandatory)
.. *Source Repository*: URL of the repository where the software code is saved (mandatory)
.. *Branch*: application code branch in the repository (mandatory)
.. *Root path*: [get info] (optional/mandatory)
. Click *Binary* and enter the following fields.
.. *Group*: [get info] (optional/mandatory)
.. *Artifact*: [get info] (optional/mandatory)
.. *Version*: software version of the application
.. *Packaging*: [get info] (optional/mandatory)
. Click *Create*. The dialog closes and the new application appears in the list of defined applications.

[id="mta-assessment-open-app_{context}"]
== Opening an existing application

This procedure describes how to open and edit an application that has already been defined in {ProductShortName}.

.Prerequisites

* Log into a server running {ProductShortName}

.Procedure

> .MTA application inventory

Author: ```suggestion

```

. In the form, supply the following information:
.. *Name*: unique name for the new application (mandatory)
.. *Description*: short description of the application (optional)
.. *Business service*: purpose of the application (optional)
.. *Tags*: software tags that characterize the application (mandatory)
.. *Owner*: registered software owner from drop-down list (optional)
.. *Contributors*: contributors from drop-down list (optional)
.. *Comments*: relevant comments on the migrated application
. Click *Source code* and enter the following fields.
.. *Repository type*: *Git* or *Subversion* (mandatory)
.. *Source Repository*: URL of the repository where the software code is saved (mandatory)
.. *Branch*: application code branch in the repository (mandatory)
.. *Root path*: [get info] (optional/mandatory)
. Click *Binary* and enter the following fields.
.. *Group*: [get info] (optional/mandatory)
.. *Artifact*: [get info] (optional/mandatory)
.. *Version*: software version of the application
.. *Packaging*: [get info] (optional/mandatory)
. Click *Create*. The dialog closes and the new application appears in the list of defined applications.

> :_content-type: PROCEDURE

Author: ```suggestion

```

* Assessed applications are grouped into *archetypes* with similar application characteristics.
* Assessment module integrates tightly with the application inventory.
* Assessment questionnaires can be imported and exported.
* Enhanced questionnaire capabilities:
** Conditional questions
** Application auto-tagging according to answers
** Automated answers from tags in application archetypes

In the MTA user interface, the Assessment Module appears in a separate menu item. Selecting the item shows the currently saved applications.

The {ProductName} ({ProductShortName}) assesses applications that are to be migrated, according to a set of questions relevant to the application, such as dependencies, *[more question types]*. 
// {ProductShortName} comes with a default questionnaire/s that the user can edit, customize and save as new questionnaires.

[id="mta-assessment-new-app_{context}"]
== Creating a new application

This procedure described how to create a new application in {ProductShortName} that is not based on the settings of an existing application.

.Prerequisites

* Log into a server running {ProductShortName}

.Procedure

> // Get updated image for MTA

Author: ```suggestion

```

// Module included in the following assemblies:
//
// * docs/web-console-guide/master.adoc
// * topics/mta-assessment-module.adoc

:_content-type: REFERENCE
[id="mta-web-assessment-module_{context}"]
= Assessment Module

> The assessment process for applications that are to be migrated in Red Hat Migration Toolkit for Applications (MTA) has been separated from other tool features.

Author: ```suggestion
The assessment process for applications that are to be migrated in {ProductFullName} has been separated from other tool features.
```

* A separate assessment hub includes the assessment module.
* Assessment supports multiple questionnaires, relevant to each assessed application.
* Default questionnaires can be customized and saved.
* Users can design and version custom questionnaires and questions using YAML syntax.
* Assessed applications are grouped into *archetypes* with similar application characteristics.
* Assessment module integrates tightly with the application inventory.
* Assessment questionnaires can be imported and exported.
* Enhanced questionnaire capabilities:
** Conditional questions
** Application auto-tagging according to answers
** Automated answers from tags in application archetypes

In the MTA user interface, the Assessment Module appears in a separate menu item. Selecting the item shows the currently saved applications.

The {ProductName} ({ProductShortName}) assesses applications that are to be migrated, according to a set of questions relevant to the application, such as dependencies, *[more question types]*. 
// {ProductShortName} comes with a default questionnaire/s that the user can edit, customize and save as new questionnaires.

> :_content-type: PROCEDURE

Author: ```suggestion

```

.. *Owner*: registered software owner from drop-down list (optional)
.. *Contributors*: contributors from drop-down list (optional)
.. *Comments*: relevant comments on the migrated application
. Click *Source code* and enter the following fields.
.. *Repository type*: *Git* or *Subversion* (mandatory)
.. *Source Repository*: URL of the repository where the software code is saved (mandatory)
.. *Branch*: application code branch in the repository (mandatory)
.. *Root path*: [get info] (optional/mandatory)
. Click *Binary* and enter the following fields.
.. *Group*: [get info] (optional/mandatory)
.. *Artifact*: [get info] (optional/mandatory)
.. *Version*: software version of the application
.. *Packaging*: [get info] (optional/mandatory)
. Click *Create*. The dialog closes and the new application appears in the list of defined applications.

:_content-type: PROCEDURE
[id="mta-assessment-open-app_{context}"]
== Opening an existing application

This procedure describes how to open and edit an application that has already been defined in {ProductShortName}.

.Prerequisites

* Log into a server running {ProductShortName}

.Procedure

> .MTA application inventory[#img-mta-application-inventory]

Author: ```suggestion
.MTA application inventory
```

image::mta-assessment-apps-01.png[]

. In the {ProductShortName} user interface, select the *Migration* working mode.
. Click *Application Inventory* in the left menu bar. A list of all the available applications appears in the main pane. 

> . Click image:mta-web-icon-edit.png[*Edit*] to open the application settings, and review the application settings. See procedure above for a list of application settings.

Author: change to correct format

> . Click image:mta-web-icon-edit.png[*Edit*] to open the application settings, and review the application settings. See procedure above for a list of application settings.

Author: ```suggestion
. Click `edit` {edit} to open the application settings, and review the application settings. See procedure above for a list of application settings.
```

* Assessment module integrates tightly with the application inventory.
* Assessment questionnaires can be imported and exported.
* Enhanced questionnaire capabilities:
** Conditional questions
** Application auto-tagging according to answers
** Automated answers from tags in application archetypes

In the MTA user interface, the Assessment Module appears in a separate menu item. Selecting the item shows the currently saved applications.

The {ProductName} ({ProductShortName}) assesses applications that are to be migrated, according to a set of questions relevant to the application, such as dependencies, *[more question types]*. 
// {ProductShortName} comes with a default questionnaire/s that the user can edit, customize and save as new questionnaires.

:_content-type: PROCEDURE
[id="mta-assessment-new-app_{context}"]
== Creating a new application

This procedure described how to create a new application in {ProductShortName} that is not based on the settings of an existing application.

.Prerequisites

* Log into a server running {ProductShortName}

.Procedure

// Get updated image for MTA

> [MTA application inventory]image::mta-assessment-create-app-01.png

Author: ```suggestion
[MTA application inventory]image::mta-assessment-app-create-01.png
```

.. *Owner*: registered software owner from drop-down list (optional)
.. *Contributors*: contributors from drop-down list (optional)
.. *Comments*: relevant comments on the migrated application
. Click *Source code* and enter the following fields.
.. *Repository type*: *Git* or *Subversion* (mandatory)
.. *Source Repository*: URL of the repository where the software code is saved (mandatory)
.. *Branch*: application code branch in the repository (mandatory)
.. *Root path*: [get info] (optional/mandatory)
. Click *Binary* and enter the following fields.
.. *Group*: [get info] (optional/mandatory)
.. *Artifact*: [get info] (optional/mandatory)
.. *Version*: software version of the application
.. *Packaging*: [get info] (optional/mandatory)
. Click *Create*. The dialog closes and the new application appears in the list of defined applications.

:_content-type: PROCEDURE
[id="mta-assessment-open-app_{context}"]
== Opening an existing application

This procedure describes how to open and edit an application that has already been defined in {ProductShortName}.

.Prerequisites

* Log into a server running {ProductShortName}

.Procedure

> // Get updated image for MTA

Author: ```suggestion

```

// Module included in the following assemblies:
//
// * docs/web-console-guide/master.adoc
// * topics/mta-assessment-module.adoc

:_content-type: REFERENCE
[id="mta-web-assessment-module_{context}"]
= Assessment Module

The assessment process for applications that are to be migrated in {ProductFullName} has been separated from other tool features.

> 

Reviewer: Nothing is migrated in MTA. "The assessment process for applications has been separated from other tool features." [Is this true?]

In the {ProductShortName} user interface, the Assessment Module appears in a separate menu item. Selecting the item shows the currently saved applications.

The {ProductShortName} assesses applications that are to be migrated, according to a set of questions relevant to the application, such as dependencies, *[more question types]*.

> 

Reviewer: Something is missing here.

[id="mta-assessment-new-app_{context}"]
== Creating a new application

This procedure described how to create a new application in {ProductShortName} that is not based on the settings of an existing application.

> 

Reviewer: You can create a new application in {ProductShortName}.

.Prerequisites

* Log into a server running {ProductShortName}

> 

Reviewer: * Be logged on to a server...

.Procedure

> * In the {ProductShortName} user interface, select the *Migration* working mode.

Reviewer: "... select * Migration *" [no spaces]

* Click *Application Inventory* in the left menu bar.

> * Click *Create new*. A *New application* dialog appears.

Reviewer: The * New ..."

* In the form, supply the following information:
** *Name*: unique name for the new application (mandatory)

> ** *Description*: short description of the application (optional)

Reviewer: we don't usually specify (mandatory)

** *Business service*: purpose of the application (optional)
** *Tags*: software tags that characterize the application (mandatory)
** *Owner*: registered software owner from drop-down list (optional)
** *Contributors*: contributors from drop-down list (optional)
** *Comments*: relevant comments on the migrated application
* Click *Source code* and enter the following fields.

> ** *Repository type*: *Git* or *Subversion* (mandatory)

Reviewer: The sounds like you have to fill in every field -- aren't you choosing between opens with appropriate fields opening for each?

** *Source Repository*: URL of the repository where the software code is saved (mandatory)
** *Branch*: application code branch in the repository (mandatory)
** *Root path*: [get info] (optional/mandatory)
* Click *Binary* and enter the following fields.
** *Group*: [get info] (optional/mandatory)
** *Artifact*: [get info] (optional/mandatory)
** *Version*: software version of the application
** *Packaging*: [get info] (optional/mandatory)
* Click *Create*. The dialog closes and the new application appears in the list of defined applications.

[id="mta-assessment-open-app_{context}"]
== Opening an existing application

> This procedure describes how to open and edit an application that has already been defined in {ProductShortName}.

Reviewer: == Editing an existing application
You can edit an application...

.Prerequisites

* Log into a server running {ProductShortName}

.Procedure

image::mta-assessment-apps-01.png[MTA Application Inventory]

. In the {ProductShortName} user interface, select the *Migration* working mode.
. Click *Application Inventory* in the left menu bar. A list of all the available applications appears in the main pane.
. Click `edit` {edit} to open the application settings, and review the application settings. See procedure above for a list of application settings.

:_content-type: PROCEDURE
[id="mta-assessment-run_{context}"]
== Running an assessment on an application

> 

Reviewer: We need a description here -- and what's not clear to me is if we run can an assessment on an archetype or we always run assessments on applications, which get built up into archetypes.

.Prerequisites

* Log into a server running {ProductShortName}

> 

Reviewer: Be logged on to a server...
End with a period.

.Procedure

> * In the {ProductShortName} user interface, select the *Migration* working mode.

Reviewer: . not *

* Click *Application Inventory* in the left menu bar. A list of all the available applications appears in the main pane.
* Select an application from the list that appears in the main pane, click the Options menu {kebab} at the end of the row, and select *Assess* from the drop-down menu.

image::mta-assessment-select-questionnaire-01.png[Assessment questionnaires]

[start=4]
. *Assessment Action* appears in the main pane. Select one of the questionnaires and click *Take*.

[start=5]
. *Application assessment*

> 

Reviewer: What now? Is this supposed to lead into "MTA questionnaire" (mta-introduction-questionnaire.adoc, I think)?
